sympy/matrices/sparse.py

```diff
@@ -133,7 +133,7 @@ def __new__(cls, *args, **kwargs):
                         value = self._sympify(
                             op(self._sympify(i), self._sympify(j)))
                         if value:
-                            self._smat[(i, j)] = value
+                            self._smat[i, j] = value
```

Comment:
Personally, I have to do a double take to realize `dic[a,b]` is implicitly converted to `dic[(a,b)]`, but I defer to sympy's formatting guidelines.

Reply:
It now mirrors how one would access the matrix itself (e.g. `ones(2)[1,1]`).